runtime/src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js

```diff
@@ -347,25 +347,6 @@ it('Verify how many work item type exists in drop down - desktop', function() {
   });
 
   /*Verfify on selecting workitem it should display in list and detail view both pages */
-  it('Verfify on selecting workitem state it should display in list and detail view both pages -phone ', function() {
```

Comment:
@ldimaggi Why delete this test?

Reply:
This test - and similar tests should be removed and perhaps recreated. What we want to create as a replacement is a test to verify dynamic changes. Also - the test as written hard codes a reference to an arry element with a string used to define an icon. This is a very brittle approach.

Once the current spring ends - we should review ALL the existing tests and decide which to retain, which to remove, which to update, and which to totally recreate. Thx!

Reply:
Right :+1: